src/ape/pytest/fixtures.py

```diff
@@ -157,7 +157,7 @@ def __exit__(self, exc_type, exc_val, exc_tb):
 
     def capture_range(self, start_block: int, stop_block: int):
         blocks = self.chain_manager.blocks.range(start_block, stop_block + 1)
-        transactions = [t for b in blocks for t in b.transactions if t.receiver and t.sender]
+        transactions = [t for b in blocks for t in b.transactions if t.sender]
```

Comment:
might be able to remove `if t.sender` as well, not sure why that is there... probably preventing impersonated TXNs from working actually... hmmmm..

- [ ] check this out